edit/buildozer_test.go

```diff
@@ -14,9 +14,14 @@ distributed under the License is distributed on an "AS IS" BASIS,
 package edit
 
 import (
+	"io/ioutil"
+	"os"
+	"path/filepath"
 	"strings"
 	"testing"
 
```

Comment:
Please remove the empty line, also I believe `gofmt` will sort the imports lexicographically (except the last github import because it's separated by an empty line), so please also do that too.

Reply:
Fixed! I was assuming that `goimports`/`gofmt` would take care of all the formatting but it doesn't remove empty lines if I accidentally split the imports into sections.

Reply:
Empty lines are allowed for separating blocks of imports, like here imports from built-in libraries are separated from the github import below.